.github/workflows/ci.yml

```diff
@@ -68,8 +68,7 @@ jobs:
         run: poetry run black --check .
 
       - name: Lint
-        run: poetry run pylint ./**/*.py
-
+        run:  poetry run pylint $(git ls-files '*.py')
```

Comment:
I did not get the intended results with  `pylint --recursive=y .` Somehow that resulted in lots of errors from python libraries while the files in `learning_resources` were ignored. This lints all .py files committed in git and is the  result we want.